ansible/main.yml

```diff
@@ -30,6 +28,8 @@
       include_tasks: "tasks/database.yml"
     - name: "start process of installing phpmyadmin"
       include_tasks: "tasks/phpmyadmin.yml"
+    - name: "start process of configuring the webserver"
```

Comment:
FYI: The digidecs task also installs composer and some PHP stuff, that might be influenced by moving this

Reply:
Yeah, because of the switch PHP is now effectively installed during the digidecs tasks, instead of the nginx tasks. Doesn't pose problems though.